### PR TITLE
Bugfix director search

### PR DESCRIPTION
**Pull request by nitheesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/2628
https://github.com/bcgov/entity/issues/2694

*Description of changes:*
bugfixes:
- An owner can add 0 team members which is incorrect- ideally this add button should be disabled initially if there is no member added
- Member can also see the team member screen - which is incorrect ( I changed the role of existing admin to member)
- owner can navigate to Manage businesses tab which is incorrect
- on logout - user should be redirected to the Director Search Login page
- Cancel button is not working for create user profile
- hide "Create a new BC Registries account" button in home page

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/CreateUserProfileForm.vue [hunk 1]
```diff
@@ -191,7 +191,7 @@ export default class CreateUserProfileForm extends Mixins(NextPageMixin) {
     }
 
     private cancel () {
-      window.history.back()
+      this.$router.push('/signin/bcros/')
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/CreateUserProfileForm.vue:194]: where does it navigate if the user is signed in?

File: auth-web/src/router.ts [hunk 2]
```diff
@@ -109,7 +111,7 @@ export function getRoutes (): RouteConfig[] {
     { path: '/returnpayment/:paymentId/transaction/:transactionId', name: 'returnpayment', component: PaymentReturnView, props: mapReturnPayVars, meta: { requiresAuth: false, requiresProfile: true } },
     { path: '/searchbusiness', name: 'searchbusiness', component: SearchBusinessView, props: true, meta: { requiresAuth: true, allowedRoles: [Role.Staff] } },
     { path: '/unauthorized', name: 'unauthorized', component: UnauthorizedView, props: true, meta: { requiresAuth: false } },
-    { path: '/unauthorizedtermsdecline', name: 'unauthorizedtermsdecline', component: UnauthorizedView, props: true, meta: { requiresAuth: true } },
+    { path: '/unauthorizedtermsdecline', name: 'unauthorizedtermsdecline', component: TermsOfServiceDeclineView, props: true, meta: { requiresAuth: true } },
```
Comment (saravanpa-aot) [inline auth-web/src/router.ts:114]: I hope this view is not used anywhere else

Comment (nitheesh-aot) [reply auth-web/src/router.ts:114]: nope, this view is only for the terms decline view, earlier I reused Unauthorized page conditionally for this, but it seems like there are a lot of different conditions needed in that, thats why moved to a new view page

File: auth-web/src/components/auth/CreateUserProfileForm.vue (hunk 1 above)
Comment (nitheesh-aot) [reply auth-web/src/components/auth/CreateUserProfileForm.vue:194]: we can navigate to the home page if any other user is already logged in, also will that case happen in the real users?

**Pull request merged**